### DISCUSSION TITLE
Explicit Resource Management

### DISCUSSION THREAD

**Discussion by joelday** [nodejs/help#4742] [Q&A] [answered]
Any plan/timeline to implement https://github.com/tc39/proposal-explicit-resource-management?

**Comment by bnoordhuis** [comment]
You're asking in the wrong place. New syntax gets implemented by V8, not Node.js.

**Comment by cjihrig** [comment, marked as answer]
Like Ben said, Node will get this once V8 implements it and Node updates to a version of V8 that supports it. It's worth noting that there has already been some work to add `Symbol.dispose` and `Symbol.asyncDispose` to some Node core APIs so that transpilers can start leveraging it.
